packages/protocol/contracts/L2/TaikoL2.sol

```diff
@@ -104,13 +104,12 @@ contract TaikoL2 is AddressResolver, ReentrancyGuard, IHeaderSync {
         require(tx.gasprice == 0, "L2:gasPrice");
 
         TaikoData.Config memory config = getConfig();
-        LibInvalidTxList.Reason reason = LibInvalidTxList.isTxListInvalid({
+        LibInvalidTxList.assertTxListInvalid({
```

Comment:
if we rename the function live this, we should not return any value from `assertTxListInvalid`.